platform/core/src/capabilities/InstantiationCapability.js

```diff
@@ -45,19 +48,26 @@ define(
          * have been persisted, nor will it have been added to the
          * composition of the object which exposed this capability.
          *
+         * @param {object} the model for the new domain object
          * @returns {DomainObject} the new domain object
          */
         InstantiationCapability.prototype.instantiate = function (model) {
+            var parsedId =
+                    this.identifierService.parse(this.domainObject.getId()),
+                space = parsedId.getDefinedSpace(),
```

Comment:
Idea here is to inherit identifiers that look like a parent's identifier; if your parent is `some-space:some-id`, you become `some-space:some-new-id`, while if your parent is just `some-id` you become just `some-new-id`. This seems semantically consistent (an unprefixed ID means "use the configured default space at run-time" and a prefixed ID means "use this specific space".) Configured default spaces may change over time, but persisted identifiers don't. If we persist these, changes to the literal string that names the default space require migrations.

Put another way, omitting the default means that single-persistence-space applications just never need to worry about this. This is also useful for limiting the impact if we ever change the way persistence spaces get encoded (deployments that weren't using that feature are not effected.)